CONTRIBUTING.md

```diff
@@ -73,12 +73,12 @@ A template plugin can be found in (plugins/TEMPLATE.nix)[https://github.com/nix-
 | **isColorscheme**            | Indicating whether the plugin is a colorscheme.                                                                                                                                                                                                      | No       | `false`                                                                                         |
-| **luaName**                  | The Lua name for the plugin.                                                                                                                                                                                                                         | No       | `name` parameter                                                                                |
+| **moduleName**                  | The Lua name for the plugin.                                                                                                                                                                                                                         | No       | `name` parameter                                                                                |
 | **maintainers**              | Maintainers for the plugin.                                                                                                                                                                                                                          | No       | `[]`                                                                                            |
 | **optionsRenamedToSettings** | Options that have been renamed and move to the `settings` attribute.                                                                                                                                                                                 | No       | `[]`                                                                                            |
-| **originalName**             | The original name of the plugin, usually the plugin's github repo name. E.g. `"foo-bar.nvim"`.                                                                                                                                                       | No       | `name` parameter                                                                                |
+| **packPathName**             | The name of the plugin directory in [packpath](https://neovim.io/doc/user/options.html#'packpath'), usually the plugin's github repo name. E.g. `"foo-bar.nvim"`.                                                                                                                                                       | No       | `name` parameter                                                                                |
```

Comment:
Same

Reply:
Still additional whitespace after the desc. Almost looks like you have a vim conceal level enabled and the hyperlink syntax is being hidden visually while you're checking alignment?

Reply:
Sorry, didn't see the stuff that far over :P